### ISSUE TITLE
Program clarification

### ISSUE THREAD

**Issue by HurricanKai** [dotnet-foundation/project-maturity-model#4]
According to <https://devblogs.microsoft.com/dotnet/joining-the-net-foundation-maturity-model-pilot/> there is a "new pilot program" but I can't seem to find what that means? later in that same post "The .Net Foundation is proposing". 
What actually is happening? Is this "proposal" put into place? part of it?
I can't seem to find anything new on https://opensource.microsoft.com/
But part of what's proposed here is already implemented there (for example "Run an ecosystem-wide "up-for-grabs" or "help-wanted" board for projects.", though just in part).

**Comment by richlander** [comment]
Excellent questions ...

The pilot program is mostly focused on the maturity ladder. That's the part we're starting now. We are starting with a half-dozen projects and are looking for others to express interest. There are no projects officially placed on the ladder at this point. This is the next step.

The other things, like the project board is still coming. That's currently just an idea and not a concrete plan.

The entire proposal is a .NET Foundation not Microsoft initiative. So, the only thing that might show up on a Microsoft site is saying "hey, we're participating" but not defining anyway. This blog post from the .NET Team (from me, actually) is a good example: https://devblogs.microsoft.com/dotnet/joining-the-net-foundation-maturity-model-pilot/

Does that help?

**Comment by charlesroddie** [comment]
Can open source projects that are currently in a Microsoft-affiliated organization join the maturity model and/or foundation, or is this only for "community" projects?

**Comment by richlander** [comment]
All individuals and organizations are welcome to participate provided their projects satisfy the characteristics in the maturity model. So, yes, Microsoft is invited to participate.

**Comment by HurricanKai** [comment]
I see, is there a list of Participating Projects yet (Level 4 / 3 / 2 / 1?) And is there a reason all dotnet/ projects aren't automatically level 3 (4?) I believe all of them meet the requirements, and it would quickstart things, since further levels also require dependencies to be of the same (or higher) level. Ie all Rx dependent projects need Rx to be approved at some point (if they want to be approved themselves)

**Comment by clairernovotny** [comment]
Right now, there isn't a centralized list of projects at each level, but that's a great idea. We'll need an underlying list to power the badge server and it should be exposed on a site as well.

Existing .NET Foundation projects will be encouraged to join at level three, but we can't force them to. Most of them already meet the criteria given that there's been a reasonably high bar for projects to join the Foundation already.

**Comment by richlander** [comment]
Right. This is all new, so we're starting off with a clean slate. We'll start registering projects shortly, and get the badge server running.

**Comment by jongalloway** [comment]
@HurricanKai It looks like everything's been answered here, okay if we close this issue?
